WORKSPACE

```diff
@@ -12,12 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-git_repository(
+http_archive(
```

Comment:
I'm a fan of using http_archive over git_repository for speed and security, but we should be mindful when updating versions that it's more than just a tag update, but rather checking archive name, sha'ing, and updating the prefix.

Reply:
Agreed that the overhead is a bit annoying. [Internal](http://cl/173579337) and [external](https://docs.bazel.build/versions/master/be/workspace.html#git_repository) guidance implies that we may want/need to move over to http_archive unfortunately.

Reply:
makes sense!